app/helpers/application_helper.rb

```diff
@@ -79,8 +79,8 @@ def multi_conditional_field(f, property, enablers)
       label_html: { class: 'conditional-group' }
   end
 
-  def confirm_deleting(record)
-    if ['assignments', 'journals'].include? controller_name
+  def confirm_deleting(record, controller_name)
```

Comment:
I pass it down, so as to overwrite it. I might have to find another way to customize the deletion messages.